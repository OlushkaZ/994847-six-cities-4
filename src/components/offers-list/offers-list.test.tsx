import React from 'react';
import renderer from 'react-test-renderer';

import OffersList from './offers-list';

it(`Should render OffersList correctly`, () => {
  const tree = renderer
    .create(
        <OffersList offers={[]} />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
