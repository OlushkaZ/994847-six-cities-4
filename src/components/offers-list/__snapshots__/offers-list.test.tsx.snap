// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render OffersList correctly 1`] = `
<div
  className="cities__places-list places__list tabs__content"
/>
`;
